quickstarts/quarkus-with-helm.yaml

```diff
@@ -0,0 +1,162 @@
+﻿apiVersion: console.openshift.io/v1
+kind: ConsoleQuickStart
+metadata:
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: 'true'
+    include.release.openshift.io/self-managed-high-availability: 'true'
+    include.release.openshift.io/single-node-developer: 'true'
+  name: quarkus-with-helm
+spec:
+  conclusion: >-
+    Your Quarkus application is deployed and ready!
+  description: 'Deploy a Quarkus Application from a Helm Chart.'
+  displayName: Get started with Quarkus with a Helm Chart
```

Comment:
If the phrase "Get started _with_ X _with_ a Helm Chart" is standard, then cool, but the duplication triggered me and I think it may be better to be "Get started with Quarkus _using_ a Helm Chart"